recipes/limitcpu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.6" %}
+
+package:
+  name: limitcpu
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/limitcpu/files/limitcpu/cpulimit-{{ version }}.tar.gz
+  sha256: 2f8ceeb9de43a6de52ce50e67d618c715e714b72d806f3730b5661d5bfc9c075
+  patches:
+    - 0001-Fix-__APPLE_-typo.patch
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+  script:
+    - make install CFLAGS+=' -DVERSION=$(VERSION)'
```

Comment:
Do you not need to run configure with the new PREFIX?

Reply:
Nope, only a single `Makefile` that is really quiet pleasant for us and uses the environment's `CC`/`*FLAGS`/`PREFIX` as we need it.